test_chdir fails

test_chdir fails on Mac OS X (Yosemite) with the following stacktrace

<pre>
__________________________________ test_chdir ___________________________________

    def test_chdir():
        curr_dir = os.getcwd()
        try:
            temp_dir = tempfile.mkdtemp()
            with utils.chdir(temp_dir):
                new_dir = os.getcwd()
>           assert new_dir == temp_dir
E           assert '/private/var...T/tmp79qx1gjm' == '/var/folders/...T/tmp79qx1gjm'
E             - /private/var/folders/nq/fx4vf3v14jz9b8mldvcyv1xr0000gn/T/tmp79qx1gjm
E             ? --------
E             + /var/folders/nq/fx4vf3v14jz9b8mldvcyv1xr0000gn/T/tmp79qx1gjm

tests/test_utils.py:20: AssertionError
</pre>

--- a/tests/test_utils.py
+++ b/tests/test_utils.py
@@ -17,7 +17,7 @@ def test_chdir():
         temp_dir = tempfile.mkdtemp()
         with utils.chdir(temp_dir):
             new_dir = os.getcwd()
-        assert new_dir == temp_dir
+        assert new_dir == os.path.realpath(temp_dir)
         assert curr_dir == os.getcwd()
     finally:
         os.rmdir(temp_dir)

